index.bs

```diff
@@ -761,6 +764,9 @@ controls how many times a single [=attribution source=] whose
 [=attribution source/source type=] is "<code>[=source type/event=]</code>" can create an
 [=event-level report=].
 
+<dfn>Max aggregatable reports per source</dfn> is a positive integer that controls how many times a
+single [=attribution source=] can create an [=aggregatable report=].
```

Comment:
I might phrase this differently, as the source doesn't really create the report. The report is rather the result of attribution.

Reply:
Rephrased a little bit, wdyt? Thanks.